systemtest/cmd/sendotlp/main.go

```diff
@@ -122,7 +199,7 @@ func Main(ctx context.Context) error {
 	if err := otlpTraceExporter.Shutdown(ctx); err != nil {
 		return err
 	}
-	return nil
+	return grpcConn.Close()
```

Comment:
is it OK to close the grpcConn twice? I guess we'll just receive a `ErrClientConnClosing` on the `defer` call and that won't be returned.

Reply:
Yes, it's safe - that is what will happen.